packages/manager/modules/dbaas-logs/src/logs/translations/Messages_fr_FR.json

```diff
@@ -51,16 +51,48 @@
+  "logs_welcome_description_title_2": "Tarification claire",
+  "logs_welcome_description_2": "Bénéficiez d'une solution de gestion des logs complète et à la demande. Pour une utilisation avancée, il vous suffit d’ajouter des options à votre plateforme. Et grâce à des prix prévisibles ainsi qu’à un trafic illimité, vous ne serez jamais surpris par votre facture mensuelle. D’ailleurs, vous profitez de seuils et d’alertes pour optimiser le suivi de votre consommation.",
   "logs_welcome_try": "Commander",
+  "logs_welcome_guides_overview_title": "Présentation de Logs Data Platform",
+  "logs_welcome_guides_quick_start_title": "Utiliser Logs Data Platform",
+  "logs_welcome_guides_community_title": "Échanger sur Logs Data Platform",
   "logs_welcome_graylog_webui": "Interface web Graylog",
   "logs_welcome_grafana_saas": "Grafana SaaS",
   "logs_welcome_kibana_supported": "Compatible Kibana",
   "logs_welcome_product_page": "Page produit",
   "logs_welcome_documentation": "Documentation",
   "logs_welcome_forum": "Forum",
+  "logs_order_title": "Création de compte Logs Data Platform",
+  "logs_order_subtitle": "Créez votre compte Logs Data Platform pour accéder à toutes les fonctionnalités",
+  "logs_order_selection_title": "Choisissez l’option Logs Data Platform qui correspond à vos besoins",
+  "logs_order_selection_subtitle": "Gérez, sécurisez et ne payez que ce que vous utilisez",
+  "logs_order_selection_details_standard_title": "Standard",
+  "logs_order_selection_details_standard_description": "Inclus gratuitement et de façon illimitée :",
+  "logs_order_selection_details_standard_description_1": "tableau de bord et flux Graylog",
+  "logs_order_selection_details_standard_description_2": "contrôle d’accès à base de rôles",
+  "logs_order_selection_details_standard_description_3": "alias Elasticsearch",
+  "logs_order_selection_details_standard_description_4": "appels API Graylog et Elasticsearch",
+  "logs_order_selection_details_standard_description_5": "alertes",
+  "logs_order_selection_details_standard_description_6": "suivi des logs en temps réel.",
+  "logs_order_selection_details_standard_price": "Les 100 premiers Go : 0,99 € HT/Go/mois",
+  "logs_order_selection_details_standard_price_ext": "puis 0,39 € HT/Go/mois",
+  "logs_order_selection_details_enterprise_title": "Enterprise",
+  "logs_order_selection_details_enterprise_description": "En plus de l’offre Standard, cette option comprend :",
+  "logs_order_selection_details_enterprise_description_1": "une liste de contrôle d'accès (ACL) réseau",
+  "logs_order_selection_details_enterprise_description_2": "la possibilité de récupérer les logs du répartiteur de charge",
+  "logs_order_selection_details_enterprise_description_3": "de la rétention personnalisable, sur demande",
+  "logs_order_selection_details_enterprise_price": "200 € HT/mois",
```

Comment:
could we fetch prices from the catalog ?

Reply:
done :)